index.js

```diff
@@ -1,18 +1,44 @@
-/***** Deliverable 1 *****/
-const header = document.querySelector("#header")
+/***** Deliverable 1 *****/ //[DONE]
+const header = document.querySelector("h1#header")
 console.log("Here's your header:", header)
 
 
-/***** Deliverable 2 *****/
+/***** Deliverable 2 *****/ //[DONE]
 
+header.style.color = "blue"
 
+/***** Deliverable 3 *****/ //[DONE]
+console.log('This is what the traveler object looks like: ', traveler)
+const profile = document.getElementById("profile")
 
-/***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+const image = profile.getElementsByTagName("img")[0]
+image.src = traveler.photo
+image.alt = traveler.name
 
+const h2 = profile.getElementsByTagName("h2")[0]
+h2.innerHTML = traveler.name
 
-/***** Deliverable 4 *****/
+const em = profile.getElementsByTagName("em")[0]
+em.innerHTML = traveler.nickname
 
+/***** Deliverable 4 *****/ //[DONE]
 
+const animal_sightings = traveler.animalSightings
+const ul = document.querySelector("ul#animals")
 
-/***** Deliverable 5 *****/
+
+animal_sightings.forEach(function temp(animal_sight){
+    const newDiv = document.createElement("div")
+    newDiv.innerHTML = `<li data-id="${animal_sight.id}">
+    <p>${animal_sight.description}</p>
+    <img src="${animal_sight.photo}" alt="${animal_sight.species}"/>
+    <a href="${animal_sight.link}" target="_blank">Here's a video about the ${animal_sight.species} species!</a>
+  </li>`
+  ul.appendChild(newDiv)
+    
+})
+
+/***** Deliverable 5 *****/ //[DONE]
+
+const wanted = document.querySelector("div [data-id='3']")
+wanted.remove()
```

Comment:
good! funny variable name haha 

Just suggesting a css selector update that would tie in with my previous suggestions. 

```suggestion
const wanted = document.querySelector("li[data-id='3']")
wanted.remove()
```